src/main/util.hpp

```diff
@@ -0,0 +1,19 @@
+#include <map>
+#include <string>
+
+namespace util {
+
+enum Severity {
+    NORMAL,
+    DEBUG,
+    ERROR,
+    WARNING,
+    FATAL,
+};
+
+const char *SeverityText[] = {
+    "", "Normal: ", "Debug: ", "Error: ", "Warning: ", "Fatal: "};
```

Comment:
Why is there an empty string at the beginning? Won't this make the `SeverityText[severity]` return the wrong message?

Reply:
Do enums start at 0 or 1 by default? Thinking about it more, 0 makes more sense. I can check